internal/tools/applications.go

```diff
@@ -44,3 +43,242 @@ func GetApplicationListTool() (mcp.Tool, server.ToolHandlerFunc) {
 
 	return appListTool, appListToolImpl
 }
+
+func GetSinglePageAppTool() (mcp.Tool, server.ToolHandlerFunc) {
```

Comment:
```suggestion
func GetSinglePageAppCreationTool() (mcp.Tool, server.ToolHandlerFunc) {
```

Reply:
Added this suggestion for better readability. WDYT?

Reply:
+1. Refactored all method names. Please have a look